msbuild-sonarqube-runner/msbuild-sonarqube-runner.nuspec

```diff
@@ -4,7 +4,7 @@
   <metadata>
     <id>msbuild-sonarqube-runner</id>
     <title>MSBuild SonarQube Runner</title>
-    <version>1.0.0.20150831</version>
+    <version>1.1</version>
```

Comment:
@yetanotherchris can you make it `1.1.0.20160205` ? The reason for this is that we might have to do multiple releases of the Chocolatey package for the exact same version of the MSbuild Scanner (for example if one of the powershell script is buggy) 

AFAIK it's a best practice to use today's date